xml_converter/generators/proto_templates/node.proto

```diff
@@ -0,0 +1,274 @@
+syntax = "proto3";
+import "google/protobuf/any.proto";
+
+package Proto_Node;
+
+message Category {
+    bool default_visibility = 1;
+    string display_name = 2;
+    bool is_separator = 3;
+    string name = 4;
+    string tooltip_name = 5;
+    map <string, Category> children = 6;
+}
+
+message Icon {
+    fixed32 achievement_bit = 1;
+    int32 achievement_id = 2;
+    float alpha = 3;
+    bool can_fade = 4;
+    float distance_fade_end = 5;
+    float distance_fade_start = 6;
+    float height_offset= 7;
+    float __tentative__scale = 8;
+    int32 map_display_size = 9;
+    int32 map_id = 10;
+    int32 maximum_size_on_screen = 11;
+    int32 minimum_size_on_screen = 12;
+    bool __tentative__render_ingame = 13;
+    bool __tentative__render_on_map = 14;
+    bool __tentative__render_on_minimap = 15;
+    bool scale_on_map_with_zoom = 16;
+    string bhdraft__schedule = 17;
+    float bhdraft__schedule_duration = 18;
+    string tip_description = 19;
+    string tip_name = 20;
+    
+    message texture {
```

Comment:
Don't use nested messages here. They are not nested in the python code which means that these will just end up being global types that get duplicate nesting.

Just make them global messages.

Reply:
Reworked to avoid Nested messages